README.md

```diff
@@ -1,47 +1,16 @@
-# BepInEx LoveMachine ([日本語](マニュアル.md))
-[![.NET](https://github.com/Sauceke/BepInEx.LoveMachine/actions/workflows/commit.yml/badge.svg)](https://github.com/Sauceke/BepInEx.LoveMachine/actions/workflows/commit.yml)
-[![Download](https://img.shields.io/github/downloads/Sauceke/BepInEx.LoveMachine/total)](https://github.com/Sauceke/BepInEx.LoveMachine/releases/latest/download/LoveMachineInstaller.exe)
+# BepInEx LoveMachine ([日本語](Documentation/Japanese))
```

Comment:
This file is overwritten with doc/README.md after each release. This is how I ensure the readme always describes the _current_ capabilities of the software.
Any direct changes to this file will be lost.